mlflow/pyfunc/model.py

```diff
@@ -226,13 +226,20 @@ class ChatModel(PythonModel, metaclass=ABCMeta):
 
     See the documentation of the ``predict()`` method below for details on that parameters and
     outputs that are expected by the ``ChatModel`` API.
+
+    Warning: Before or during the 2.20 release, we will be requiring a predict_stream
```

Comment:
nit: to be consistent with the other messages

```suggestion
    Warning: Before or during the MLflow 2.20 release, we will be requiring a predict_stream
```